sousmot/sousmotapp/static/scss/main.scss

```
@@ -68,27 +68,101 @@ $navbar-item-active-color: $navbar-item-hover-color;
     }
 
     @include mobile {
-      max-width: 80vw;
+        max-width: 80vw;
     }
 }
 
 footer.footer {
     text-align: center;
-    padding:3rem 1.5rem 4rem;
+    padding: 3rem 1.5rem 4rem;
 }
 
 body {
-    display:flex;
+    display: flex;
     min-height: 100vh;
     flex-direction: column;
 }
 
 section {
-    flex:1;
+    flex: 1;
+    touch-action: manipulation;
 }
 
 .has-addons-fullwidth .button.is-fullwidth {
-    width:50%;
+    width: 50%;
+}
+
+// Game
+#timer {
+    height: 2rem;
+}
+
+.table {
+    display: inline;
+}
+
+.table td {
+    padding: 0.5rem;
+    color: $text;
+    font-size: 2rem;
+    font-weight: bold;
+    background-color: $navbar-background-color;
+}
+
+.table td div {
+    width: 50px;
+    height: 50px;
+    overflow: hidden;
+
+    @include mobile {
+        width: 35px;
+        height: 35px;
+        font-size: 1.5rem;
+    }
+}
```

```suggestion
.table {
    display: inline;
    
    td {
        padding: 0.5rem;
        color: $text;
        font-size: 2rem;
        font-weight: bold;
        background-color: $navbar-background-color;
        
        div {
            width: 50px;
            height: 50px;
            overflow: hidden;

            @include mobile {
                width: 35px;
                height: 35px;
                font-size: 1.5rem;
            }
        }
    }
}
```
You can use the SCSS syntax :wink: